main.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable linebreak-style */
+// /* eslint-disable no-undef
+// eslint-disable no-use-before-define
+/* eslint-disable linebreak-style */
+// create the book constructor
+
+function Book(title, author, isbn) {
+  this.title = title;
+  this.author = author;
+  this.isbn = isbn;
+}
+// UI constructor
+const selectors = {
+  title: 'title',
+  author: 'author',
+  isbn: 'isbn',
+};
+function UI() {}
+
+UI.prototype.addBook = function (book) {
+  const list = document.createElement('tr');
+  list.innerHTML = `
+  <td>${book.title}</td>
+  <td>${book.author}</td>
+  <td>${book.isbn}</td>
+  <td><button class="delete">Remove</button></td>
+`;
+  document.getElementById('book-list').appendChild(list);
+};
+UI.prototype.clear = function () {
+  document.getElementById(selectors.title).value = '';
+  document.getElementById(selectors.author).value = '';
+  document.getElementById(selectors.isbn).value = '';
+};
+UI.prototype.showAlert = function (message, className) {
+// create a div
+  const div = document.createElement('div');
+  div.className = `alert ${className}`;
+  // addtext
+  div.appendChild(document.createTextNode(message));
+  // insert into the dom
+  const container = document.querySelector('.container');
+  const form = document.querySelector('#book-form');
+  container.insertBefore(div, form);
+  // set the timeout
+  setTimeout(() => {
+    div.remove();
+  }, 1000);
+};
+UI.prototype.delete = function (target) {
+  if (target.className === 'delete') {
+    target.parentElement.parentElement.remove();
+  }
+};
+// local storage class
+class Store {
+  static get() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static display() {
+    const books = Store.get();
+
+    books.forEach((book) => {
+      const ui = new UI();
+
+      // add to UI
+      ui.addBook(book);
+    });
+  }
+
+  static add(book) {
+    const books = Store.get();
+
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static remove(isbn, title) {
+    const books = Store.get();
+    books.forEach((book, index) => {
+      if (book.isbn === isbn || book.title === title) {
+        books.splice(index, 1);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use of array methods such as `slice` over `splice`, which doesn't mutate the array.